Core/include/Acts/EventData/SourceLink.hpp

```diff
@@ -17,6 +17,11 @@
 #include <type_traits>
 #include <utility>
 
+#if !defined(ACTS_SOURCELINK_SBO_SIZE)
+// Do not set this in code, use CMake!
+#define ACTS_SOURCELINK_SBO_SIZE 16
```

Comment:
would it make sense to default this in `CMake`?

Reply:
I guess there's arguments on both sides. Having the default in CMake requires that the preprocessor define is always set. I think I prefer having the default in the source file, and the default being that the preprocessor define is absent.